app/serializers/export/contact_serializer.rb

```diff
@@ -1,12 +1,28 @@
 module Export
   class ContactSerializer < ActiveModel::Serializer
     attributes :sharing,
-               :receiving,
+               :following,
```

Comment:
You can add a `"description"` (doesn't hurt), but we should still document the whole migration feature somewhere. We can do that as part of the federation documentation or do an extra page at http://diaspora.github.io/migration-documentation